### PR TITLE
ref(contrib/ec2): minor cleanup in EC2 scripts

### PR DESCRIPTION
**Pull request by carmstrong**

### CODE REVIEW

**Comment by carmstrong** [comment]
Note that I stopped Jenkins since he wouldn't test this.

**Comment by mboersma** [comment]
This EC2 script tidying LGTM.

File: contrib/ec2/provision-ec2-cluster.sh [hunk 1]
```diff
@@ -44,5 +44,5 @@ aws cloudformation create-stack \
     --stack-name $NAME \
     --parameters "$(<$THIS_DIR/cloudformation.json)"
 
-echo_green "Your Deis cluster has successfully deployed to AWS CloudFormation."
-echo_green "Please continue to follow the instructions in the README."
+echo_green "Your Deis cluster has been successfully deployed to AWS CloudFormation and is currently starting."
```
Comment (bacongobbler) [inline contrib/ec2/provision-ec2-cluster.sh]: nitpick, but "been" is past tense. We're deploying in the present tense!

**Comment by bacongobbler** [comment]
code LGTM

**Pull request merged**